wsclient/src/main/java/org/hpccsystems/ws/client/HPCCFileSprayClient.java

```diff
@@ -1930,9 +1933,43 @@ public GetDFUWorkunitsResponseWrapper getDFUWorkunits(String cluster, Long pages
         return new GetDFUWorkunitsResponseWrapper(response);
     }
 
+    /**
+     * Deletes a file from a drop zone
+     *
+     * @param dropzoneName
+     *          - The name of the dropzone to delete from
+     * @param fileNames
+     *          - A List of names to delete from the drop zone
+     * @param netAddress
+     *          - The net address of the dropzone
+     * @param path
+     *          - The full path to the dropzone on the filesystem ie /var/lib/HPCCSystems/mydropzone
+     * @param {string} [os]
+     *          - The os
```

Comment:
If we use Utils.HPCCEnvOSCode  as noted below, this becomes self documenting.
Otherwise I think it's any numeric value not 0 == linux, otherwise windows?

Reply:
It seems like 2 is linux according to the Utils class

```
        MachineOsW2K (
                0
        ), MachineOsSolaris (
                1
        ), MachineOsLinux (
                2
        ), MachineOsUnknown (
                3
        );
```